Enhancement: Run "pa app remove" as admin. Closes #5519 (#5610)
pnp/cli-microsoft365: docs/docs/cmd/pa/app/app-remove.mdx

By default, the command will try to remove a Power App. As maker, you are able to delete the Power Apps you own. As administrator, you are also able to delete Power Apps from other users.

> To remove the app from another user, use the `asAdmin` option and make sure to specify the `environmentName` option.

Reviewer: ```suggestion
To remove the app you do not own, use the `asAdmin` option and make sure to specify the `environmentName` option.
```

m365 pa app remove --name 3989cb59-ce1a-4a5c-bb78-257c5c39381d --force
```

> Removes the specified Power App you don't have access to

Reviewer: ```suggestion
Removes the specified Power App you do not own
```
